recipe/meta.yaml

```diff
@@ -14,14 +14,14 @@ source:
     code-server
 
 build:
-  number: 0
+  number: 1
   binary_relocation: False  # [osx]
   skip: true  # [win]
 
 requirements:
   host:
   run:
-    - nodejs 12.*
+    - nodejs =>12
```

Comment:
Can you use `>=` here? That's the more common form of it.

Reply:
I honestly don't know what I'm doing! I just saw it in https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#build-version-spec . (See "EXAMPLE: >=2,<3")

Reply:
Ah, I didn't see that typo until now!

Reply:
Ok, I think I fixed it now. You can see my new commit, right?

Reply:
Yes, looks right